custom-widget-builder/src/CustomWidgetBuilder.ts

```diff
@@ -197,12 +269,48 @@ export class CustomWidgetBuilder {
     return `Cannot get any information from file ${wcFile}\nExiting...`;
   }
 
-  private copyFile(file: string, dir: string) {
+  private static copyFile(file: string, dir: string) {
     let sourceBuff = fs.readFileSync(file);
     let fileName = path.basename(file);
     fs.writeFileSync(`${dir}/${fileName}`, sourceBuff);
   }
 
+  private static getUUIDv4() {
+    return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
```

Comment:
Since it's only 3 lines of code, I don't think this deserve to add a dependency for this. (Note that these lines have been used and commented by several people from StackOverflow)